fakeredis.py

```diff
@@ -312,6 +312,27 @@ def renamenx(self, src, dst):
         else:
             return self.rename(src, dst)
 
+    def scan(self, cursor=0, match=None, count=None):
+        cursor = int(cursor)
+        if count is None:
+            count = 10
+        match_keys = [key for key in self._db \
+            if not key or not match or \
```

Comment:
`if not key` would exclude the empty string which I believe is a valid key name.